test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js

```diff
@@ -1123,12 +1107,12 @@ describe('Java expression language (JEXL) based transformations plugin', functio
                 .matchHeader('fiware-service', 'smartgondor')
                 .matchHeader('fiware-servicepath', 'gardens')
                 .post(
-                    '/v2/entities/gps1/attrs',
+                    '/v2/entities?options=upsert',
                     utils.readExampleFile(
                         './test/unit/ngsiv2/examples/contextRequests/updateContextExpressionPlugin32.json'
                     )
                 )
-                .query({ type: 'GPS' })
+                //uery({ type: 'GPS' })
```

Comment:
Lefotver?

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1111/commits/2fb5b110a37aacba924480c621934963462df562